### PR TITLE
Fix rspy297/pods affinity

### PR DESCRIPTION
**Pull request by clystard-csgroup**

### CODE REVIEW

File: apps/keycloak/keycloak.yaml [hunk 1]
```diff
@@ -41,4 +41,16 @@ spec:
     - name: log-level
       value: "{{ keycloak.logLevel }}"
     - name: metrics-enabled
-      value: 'true'
\ No newline at end of file
+      value: 'true'
+  unsupported:
+      podTemplate:
```
Comment (nleconte-csgroup) [inline apps/keycloak/keycloak.yaml]: Bad indentation here

File: apps/keycloak/kustomization.yaml [hunk 2]
```diff
@@ -23,3 +23,19 @@ resources:
   - servicemonitor.yaml
   - secret.yaml
   - keycloakrealmimport.yaml
+
+
+# patches:
```
Comment (nleconte-csgroup) [inline apps/keycloak/kustomization.yaml]: Remove unnecessary comments

**Review by nleconte-csgroup — CHANGES_REQUESTED** [review]

File: apps/keycloak/keycloak.yaml (hunk 1 above)
Comment (clystard-csgroup) [reply apps/keycloak/keycloak.yaml]: Typo fix

**Review by nleconte-csgroup — APPROVED** [review]

**Comment by nleconte-csgroup** [comment]
@clystard-csgroup Please go ahead with the merge :)

**Pull request merged**